src/test/resources/Homework18.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Homework18">
+
+    <test name="Example Test" preserve-order="false">
+        <classes>
+            <class name="Homework18">
+                <methods>
+                    <include name="testPlaySong"/>
+                </methods>
+            </class>
+        </classes>
+    </test>
+</suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/211189603-e7a64280-ca3d-42e5-aa8e-d08567f8df04.png)